src/feed/event.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import LiveTime from './livetime.js'
+import moment from 'moment'
+
+class Event extends React.Component {
+
+  static propTypes = {
+    subject: React.PropTypes.shape({
+      photo: React.PropTypes.string.isRequired,
+      text: React.PropTypes.string.isRequired,
+      link: React.PropTypes.string.isRequired,
+    }).isRequired,
+    created_at: React.PropTypes.string.isRequired,
+    story: React.PropTypes.string.isRequired,
+    object1: React.PropTypes.shape({
+      entity: React.PropTypes.string.isRequired,
+      text: React.PropTypes.string.isRequired,
+      link: React.PropTypes.string
+    }),
+    object2: React.PropTypes.shape({
+      entity: React.PropTypes.string.isRequired,
+      text: React.PropTypes.string.isRequired,
+      link: React.PropTypes.string
+    }),
+
+  }
+
+  static defaultProps = {
+  }
+
+  render() {
+    var description = '<a href="'+this.props.subject.link+'">'+this.props.subject.text+'</a> '
+    description += this.props.story
```

Comment:
All this string concatenation feels out of place. It appears you're templating some details of the entities which play a part in this event in the story, then injecting HTML directly. Note that if we don't 100% control the naming of these entities, this is an XSS attack vector. It hints at an anemic data model that's creating a representation implicitly rather than explicitly.